GitUpKit/Utilities/GILaunchServicesLocator.h

```diff
@@ -0,0 +1,41 @@
+//
+//  GILaunchServicesLocator.h
+//  GitUpKit (OSX)
+//
+//  Created by Dmitry Lobanov on 08.10.2019.
+//
+
+#import <Foundation/Foundation.h>
+NS_ASSUME_NONNULL_BEGIN
+extern NSString* const GIViewController_DiffTool;
+extern NSString* const GIViewController_MergeTool;
+extern NSString* const GIViewController_TerminalTool;
```

Comment:
Why `GIViewController`?

Reply:
@lucasderraugh Legacy, I guess.
`GIPreferences_` prefix is better?

```objective-c
// Change
extern NSString *const GIPreferences_DiffTool;
extern NSString *const GIPreferences_MergeTool;
extern NSString *const GIPreferences_TerminalTool;

// DiffTool and MergeTool
extern NSString *const GIPreferences_DiffMergeTool_FileMerge;
extern NSString *const GIPreferences_DiffMergeTool_Kaleidoscope;
extern NSString *const GIPreferences_DiffMergeTool_BeyondCompare;
extern NSString *const GIPreferences_DiffMergeTool_P4Merge;
extern NSString *const GIPreferences_DiffMergeTool_GitTool;
extern NSString *const GIPreferences_DiffMergeTool_DiffMerge;

// TerminalTool
extern NSString *const GIPreferences_TerminalTool_Terminal;
extern NSString *const GIPreferences_TerminalTool_iTerm;
```

Reply:
Also I would like to add suffix to distinguish `Keys` and `DisplayNames`.

```objective-c
extern NSString *const GIPreferences_DiffTool_Key;
extern NSString *const GIPreferences_DiffMergeTool_FileMerge_DisplayName;
```

Reply:
Ah, I see. I think `GIPreferences` works well. I think you can shorten up some of these names. If it's just being used as the localized string value, just `GIPreferences_FileMerge` would work fine if the string is just going to be "File Merge" anyways.

Reply:
@lucasderraugh Fixed.